packages/manager/apps/dedicated/client/app/account/otrs/otrs.html

```diff
@@ -119,22 +119,12 @@ <h1 data-translate="otrs_title"></h1>
                        data-translate="otrs_search_ticket_minCreationDate">
                 </label>
                 <div class="input-group">
-                    <span class="input-group-btn">
-                        <button type="button"
-                                class="btn btn-default"
-                                data-ng-click="minCreationDate()">
-                            <span class="fa fa-calendar"></span>
-                        </button>
-                    </span>
-                    <input type="text"
-                           class="form-control"
-                           data-ng-model="search.minCreationDate"
-                           data-ng-required="true"
-                           data-ng-focus="minCreationDate()"
-                           data-uib-datepicker-popup="dd-MMMM-yyyy"
-                           data-is-open="minCreationDateSearch.opened"
-                           data-close-text="Close"
-                           data-show-button-bar="false" />
+                    <div class="input-group-addon">
+                        <i class="fa fa-calendar"></i>
```

Comment:
Isn't that already handled by `oui-calendar`, if not maybe it should be

Reply:
There are places where the icon is not shown. For example: https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/billing/components/directives/dateRange/billingDateRange.html#L37

And there are places where we show the icon on the right side of the control. For example: https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.html#L196

Hence, in order to provide this flexibility, I feel it has to be left this way. Please let us know if you think otherwise :)

Reply:
@ganeshkumar1989 maybe a solution could be to have this icon as an option of `oui-calendar` ? That way we can easily maintain and update the behaviour 
 Moreover maybe we could use a `oui-icon` instead of a `fa-calendar`

Reply:
@marie-j , I have created a ticket to track the addition of the icon option to the `oui-calendar` component. The ticket id is UK-126

I thought I can quickly replace the fa-calendar with the oui-icon counterpart. But the size of the icon has to be reduced (icon appears bigger than the control). Instead of writing and replicating new css classes, I thought it can be done as a part of the task UK-126. Please let me know what you think

Reply:
Let's tackle this right after UK-126 ticket you've mentioned.
It means that we can create a ticket/issue that **has to be done** after UK-126.

@ganeshkumar1989 Could you please fill a new ticket? Thank you